src/openms/source/CHEMISTRY/RNaseDigestion.cpp

```diff
@@ -100,31 +144,80 @@ namespace OpenMS
     }
     else // proper enzyme cleavage
     {
-      vector<Size> fragment_pos(1, 0);
+      // with variable A->I, would enzyme cut after/before I, but not unmod. A?
+      bool cuts_after_I = (variable_inosine_ &&
+                           boost::regex_search("I", cuts_after_regex_) &&
```

Comment:
can we go without boost regex and use std regex?

Reply:
might have some speed impact though

Reply:
We use `boost::regex` consistently in the "digestion" code, so I'd rather not change it just here.
If you want to switch to `std::regex`, I think that should be a separate pull request.